catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -113,7 +114,8 @@ public void initialize(
     resultConf.put("catalog_uuid", info.id().toString());
     IcebergConfig icebergConfig = new IcebergConfig(resultConf);
 
-    this.icebergCatalogWrapper = new IcebergCatalogWrapper(icebergConfig);
+    IcebergCatalogWrapper rawWrapper = new IcebergCatalogWrapper(icebergConfig);
```

Comment:
Could we add a check to determine whether we should proxy the IcebergCatalogWrapper?